lib/glific/flows/case.ex

```diff
@@ -84,8 +84,30 @@ defmodule Glific.Flows.Case do
   Validate a case
   """
   @spec validate(Case.t(), Keyword.t(), map()) :: Keyword.t()
-  def validate(_case, errors, _flow) do
-    errors
+  def validate(cases, errors, flow) do
+    wait_for_response_words =
+      cases.arguments
+      |> List.first()
+      |> String.split(", ")
+      |> Enum.reduce(MapSet.new(), &MapSet.put(&2, &1))
```

Comment:
I think we should clean the arguments as well with Glific clean.

Reply:
@pankaj-ag Thanks for discussion. I have updated the code now